### PR TITLE
Add GLADE CMIP6 example

### PR DESCRIPTION
**Pull request by andersy005**
Adapted from the original collection residing in https://github.com/NCAR/intake-esm-datastore/blob/master/catalogs/glade-cmip6.json

### CODE REVIEW

**Comment by m-mohr** [comment]
Great, validates against the schema in #27 without any changes.

**Comment by rabernat** [comment]
Seems like this is good to go?

File: examples/glade-cmip6.json [hunk 1]
```diff
@@ -0,0 +1,137 @@
+{
+  "stac_version": "0.9.0",
+  "stac_extensions": [
+    "collection-assets",
+    "https://github.com/NCAR/esm-collection-spec/tree/master/schema.json"
```
Comment (andersy005) [inline examples/glade-cmip6.json:5]: @m-mohr, Is this going to be the final location of the schema or should I use something different? I realize that during our sprint, we agreed to use this as a placeholder.

Comment (m-mohr) [reply examples/glade-cmip6.json:5]: Should be a placeholder, It's up to you guys to figure out where you could host a schema permanently. Not sure whether you can come up with something like https://schemas.ncar.com/esm/v0.2/schema.json or so.

File: examples/glade-cmip6.json [hunk 2]
```diff
@@ -0,0 +1,137 @@
+{
+  "stac_version": "0.9.0",
```
Comment (m-mohr) [inline examples/glade-cmip6.json]: STAC 1.0.0-beta.1 has been released.
```suggestion
  "stac_version": "1.0.0-beta.1",
```

**Pull request closed without merging**